apiary.apib

```diff
@@ -2192,6 +2192,272 @@ Update a milestone.
+            + ended_at: `2017-04-26T11:01:49+00:00` (string, nullable)
+            + duration: 3600 (number) - In seconds
+            + description: `Timer description` (string)
+            + subject (object)
+                + type: `project` (enum[string])
+                    + Members
+                        + company
+                        + contact
+                        + event
+                        + todo
+                        + project
+                        + milestone
+                        + ticket
+                + id: `5883723a-b5aa-4c9d-a02b-ec0dd25f6ca2` (string)
+
+### timetracking.current [GET /timetracking.current]
+
+Get the current timetracking timer.
+
++ Response 200 (application/json)
+
+    + Attributes (object)
+        + data (object)
+            + id: `2b282dec-ba9d-4faa-9b39-944b99ee5c0a` (string)
+            + user (object)
+                + type: `user` (string)
+                + id: `5443a1b9-7673-4922-921f-1da4ff557783` (string)
+            + work_type (object)
+                + type: `workType` (string)
+                + id: `d02de940-9435-4e35-bc0b-3d17f3d0ea7b` (string)
+            + started_at: `2017-04-26T10:01:49+00:00` (string)
+            + description: `Timer description` (string)
+            + subject (object)
+                + type: `project` (enum[string])
+                    + Members
+                        + company
+                        + contact
+                        + event
+                        + todo
+                        + project
+                        + milestone
+                        + ticket
+                + id: `31931bc9-4ae8-4c50-ba8d-9ea88498c1c1` (string)
+
+### timetracking.start [POST /timetracking.start]
+
+Start a new timetracking timer.
+
++ Request (application/json)
+
+    + Attributes (object)
+        + work_type_id: `db41328a-7a25-4e85-8fb9-830baacb7f40` (string, optional)
+        + started_at: `2017-04-26T10:01:49+00:00` (string, optional) - If not provided, current time will be used
+        + description (string, optional)
+        + subject (object, optional)
+            + type: `project` (enum[string], required)
+                + Members
+                    + company
+                    + contact
+                    + event
+                    + todo
+                    + project
+                    + milestone
+                    + ticket
+            + id: `29ff471c-7d8f-40d5-8c95-9a9cab841e65` (string, required)
+
++ Response 201
+
+    + Attributes (object)
+        + data (object)
+            + type: `timetracking` (string)
+            + id: `4ff475b2-f45b-4803-800e-350b9c02aeca` (string)
+
+### timetracking.stop [POST /timetracking.stop]
+
+Stop the current timetracking timer.
+
++ Request (application/json)
+
+    + Attributes (object)
+        + ended_at: `2017-04-26T10:01:49+00:00` (string, optional) - If not provided, current time will be used
```

Comment:
Q: shouldn't this endpoint return the `id` of the stopped timetracking?

This will be different then the `id` returned from `timetracking.start`, because it's another datatype (ongoing vs finished timetracking) + the stopped timetracking can be split here when it spanned multiple days.